plugins/kjob.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kjob
+spec:
+  version: v0.1.0
+  shortDescription: Tool for CLI-loving ML researchers allowing them to run AI/ML jobs based on the pre-defined templates.
+  homepage: https://github.com/kubernetes-sigs/kjob/
+  description: |
+    This is a CLI tool implemented as a kubectl plugin. It allows ML researchers to run templated Jobs 
+    with different values for the application-specific parameters, without the need to edit and submit 
+    entire YAML files. The tool comes with built-in support for running slurm scripts inside a Kubernetes 
+    cluster.
+    
+    See the documentation for more information: https://github.com/kubernetes-sigs/kjob/blob/main/docs/_index.md.
```

Comment:
we already link to your `homepage` in the post-installation message so this is rather redundant.

Reply:
Done